analyses/snv-callers/util/tmb_functions.R

```diff
@@ -133,8 +133,7 @@ calculate_tmb <- function(tumor_sample_barcode = NULL,
     dplyr::group_by(
       #TODO: Make this column passing stuff more flexible with some tidyeval maybe
       Tumor_Sample_Barcode = tumor_sample_barcode,
-      experimental_strategy,
-      short_histology
+      experimental_strategy
```

Comment:
The associated docs here still say:
```
 # Returns:
 # A calculated total region size, and TMB for the given Tumor_Sample_Barcode,
 # returned as a single row data.frame. `experimental_strategy`, `short_histology`
 # columns are carried along.
```

That said, I'm not sure why short_histology is being removed here?

Reply:
https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1420#issuecomment-1131933494

> It's not necessary and we'd rather have folks joining `short_histology` from the appropriate histologies file, rather than using that one. 

Or put another way, one could imagine it getting out of sync with the histologies file.

Reply:
Woops my bad!!! Moving parts 🙃

Reply:
Yep I'll still update the doc. I mostly quote because I'm lazy.

Reply:
Updated the docs in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1420/commits/6e075b210df5b5b63bac84518dbde6713a13e544